jekyll-assets/css/style.css

```diff
@@ -234,13 +234,25 @@ div.subtitle p {
   margin-right: 0px;
 }
 
-#container .section {
+#container .section,
+#related-container .section {
   display: flex;
   flex-direction: column;
   width: 100%;
 }
 
-#docs-content #container {
+#related-title{
+  width: 100%;
```

Comment:
Isn't the width of the block-level `<div>` already 100% or do we need to explicitly set it?